test/integration/noderesourcetopology_cache_test.go

```diff
@@ -264,6 +268,109 @@ func TestTopologyCachePluginWithoutUpdates(t *testing.T) {
 						}).Obj(),
 			},
 		},
+		{
+			name: "GU pod: DiscardReservedNodes: allows scheduling on both Zones",
+			podDescs: []podDesc{
+				{
+					podName:      "nrt-pod-1000",
+					isGuaranteed: true,
+					resourcesMap: map[string]string{
+						string(corev1.ResourceCPU):    "16",
+						string(corev1.ResourceMemory): "24Gi",
+					},
+					expectedNode:  "fake-node-cache-1",
+					schedulerName: discardReservedSchedulerName,
+				},
+				{
+					podName:      "nrt-pod-2000",
+					isGuaranteed: true,
+					resourcesMap: map[string]string{
+						string(corev1.ResourceCPU):    "16",
+						string(corev1.ResourceMemory): "24Gi",
+					},
+					schedulerName: discardReservedSchedulerName,
+					expectedNode:  "fake-node-cache-1",
+				},
+			},
+			nodeResourceTopologies: []*topologyv1alpha2.NodeResourceTopology{
+				MakeNRT().Name("fake-node-cache-1").Policy(topologyv1alpha2.SingleNUMANodeContainerLevel).
+					Zone(
+						topologyv1alpha2.ResourceInfoList{
+							noderesourcetopology.MakeTopologyResInfo(cpu, "32", "30"),
+							noderesourcetopology.MakeTopologyResInfo(memory, "64Gi", "60Gi"),
+						}).
+					Zone(
+						topologyv1alpha2.ResourceInfoList{
+							noderesourcetopology.MakeTopologyResInfo(cpu, "32", "30"),
+							noderesourcetopology.MakeTopologyResInfo(memory, "64Gi", "62Gi"),
+						}).Obj(),
+				MakeNRT().Name("fake-node-cache-2").Policy(topologyv1alpha2.SingleNUMANodeContainerLevel).
+					Zone(
+						topologyv1alpha2.ResourceInfoList{
+							noderesourcetopology.MakeTopologyResInfo(cpu, "32", "10"),
+							noderesourcetopology.MakeTopologyResInfo(memory, "64Gi", "14Gi"),
+						}).
+					Zone(
+						topologyv1alpha2.ResourceInfoList{
+							noderesourcetopology.MakeTopologyResInfo(cpu, "32", "8"),
+							noderesourcetopology.MakeTopologyResInfo(memory, "64Gi", "10Gi"),
+						}).Obj(),
+			},
+		},
+		{
+			name: "GU pod: DiscardReservedNodes: scheduling works as expected, when deleting pods",
```

Comment:
Can we please clarify "works as expected?" :) You mean that the scheduler can make use of a node which was previously unable to accomodate a workload because it had already a workload running which consumed too many resources, and once the existing delete is issued the new workload can be scheduled without waiting for a NRT update?

If this doesn't fit in the test name/description (and I think it will hardly fit) please add it as code comment

Reply:
Sure, changing now.